src/Apps/Sell/Routes/PhoneNumberRoute.tsx

```diff
@@ -0,0 +1,111 @@
+import { COUNTRY_CODES, countries } from "Utils/countries"
+import { DevDebug } from "Apps/Sell/Components/DevDebug"
+import { Formik } from "formik"
+import { graphql, useFragment } from "react-relay"
+import { Join, PhoneInput, Spacer, Text } from "@artsy/palette"
+import { PhoneNumberRoute_me$key } from "__generated__/PhoneNumberRoute_me.graphql"
+import { PhoneNumberRoute_submission$key } from "__generated__/PhoneNumberRoute_submission.graphql"
+import { SubmissionLayout } from "Apps/Sell/Components/SubmissionLayout"
+import { SubmissionStepTitle } from "Apps/Sell/Components/SubmissionStepTitle"
+import { useSellFlowContext } from "Apps/Sell/SellFlowContext"
+import * as React from "react"
+import * as Yup from "yup"
+
+const FRAGMENT = graphql`
+  fragment PhoneNumberRoute_submission on ConsignmentSubmission {
+    userPhoneNumber {
+      display
+      regionCode
+    }
+  }
+`
+
+const FRAGMENT_ME = graphql`
+  fragment PhoneNumberRoute_me on Me {
+    internalID
+    phoneNumber {
+      countryCode
+      regionCode
+      display(format: NATIONAL)
+      originalNumber
+    }
+  }
+`
+
+const Schema = Yup.object().shape({
+  phoneNumber: Yup.string().trim(),
+})
+
+interface FormValues {
+  userPhone: string
+  phoneNumberCountryCode: string
+}
+
+interface PhoneNumberRouteProps {
+  submission: PhoneNumberRoute_submission$key
+  me: PhoneNumberRoute_me$key
+}
+
+export const PhoneNumberRoute: React.FC<PhoneNumberRouteProps> = props => {
+  const submission = useFragment(FRAGMENT, props.submission)
+  const me = useFragment(FRAGMENT_ME, props.me)
+
+  const { actions } = useSellFlowContext()
+
+  const initialValues: FormValues = {
+    userPhone:
+      submission.userPhoneNumber?.display ?? me.phoneNumber?.display ?? "",
+    phoneNumberCountryCode:
+      submission.userPhoneNumber?.regionCode ??
+      me.phoneNumber?.regionCode ??
```

Comment:
you are using the `regionCode` here instead of `countryCode` - if we start using the `countryCode`. I would be more in favor of using the country code

Reply:
we can't use the `countryCode` because the component accepts as initial value only `regionCode`

Reply:
oki - let's maybe rename `phoneNumberCountryCode` to `phoneNumberRegionCode` then